.kitchen.yml

```diff
@@ -30,3 +29,7 @@ suites:
     run_list: languages_rust::prefix
   - name: languages_ruby
     run_list: languages_ruby::default
+  - name: languages_erlang
+    run_list: languages_erlang::default
+  - name: languages_erlang_prefix
```

Comment:
Good idea @elliott-davis. I chatted with @tylercloke about this also. Let's standardize on a single fixture recipe `languages_<LANGUAGE>::default` and single integration test `integration/languages_<LANGUAGE>/serverspec/languages_<LANGUAGE>_spec.rb` per language.